tests/clar_libgit2.c

```diff
@@ -158,6 +158,18 @@ int cl_rename(const char *source, const char *dest)
 
 #endif
 
+int cl_setenv_printf(const char *name, const char *fmt, ...)
+{
+	va_list args;
+	git_buf buf = GIT_BUF_INIT;
+
+	va_start(args, fmt);
+	cl_git_pass(git_buf_vprintf(&buf, fmt, args));
+	va_end(args);
+
+	return cl_setenv(name, git_buf_cstr(&buf));
```

Comment:
Yeah, you do use that a lot.  My inclination would have been to throw it in as a static function next to the test itself though instead of as part of the text harness.